src/test/java/com/palpal/dealightbe/domain/order/presentation/OrderControllerTest.java

```diff
@@ -286,4 +295,90 @@ void updateStatus_fail() throws Exception {
 				});
 		}
 	}
+
+	@Nested
+	@DisplayName("<주문 상세 정보 조회>")
+	class detailedInfoTest {
+		String findByIdApiPath = "/api/orders/{orderId}/{memberProviderId}";
+
+		LocalDateTime createdAt = LocalDateTime.now();
+
+		OrderCreateReq orderCreateReq = new OrderCreateReq(
+			new OrderProductsReq(List.of(new OrderProductReq(1L, 3))), 1L, "도착할 때까지 상품 냉장고에 보관 부탁드려요",
+			LocalTime.of(12, 30), 10000);
+
+		OrderProductsRes productsRes = new OrderProductsRes(List.of(new OrderProductRes(1L, "달콤한 도넛", 5, 10000, 15000,
+			"https://team-08-image-bucket.s3.ap-northeast-2.amazonaws.com/donut")));
```

Comment:
저희 bucket 이름이 바뀌어서 링크만 변경해주시면 될 것 같아요!

```suggestion
		OrderProductsRes productsRes = new OrderProductsRes(List.of(new OrderProductRes(1L, "달콤한 도넛", 5, 10000, 15000,
			"https://team-08-bucket.s3.ap-northeast-2.amazonaws.com/donut")));
```